packages/liveblocks-react/src/__tests__/useInboxNotifications.test.tsx

```diff
@@ -678,43 +678,37 @@ describe("useInboxNotifications: pagination", () => {
   test("should load the next page of data when `fetchMore` is called", async () => {
     const roomId = nanoid();
 
-    const threadOne = dummyThreadData({
-      roomId,
-      createdAt: new Date("2021-01-01T00:00:00Z"),
-    });
-    const threadTwo = dummyThreadData({
-      roomId,
-      createdAt: new Date("2021-01-02T00:00:00Z"),
-    });
-    const threadThree = dummyThreadData({
-      roomId,
-      createdAt: new Date("2021-01-03T00:00:00Z"),
-    });
+    const thread1 = dummyThreadData({ roomId, createdAt: new Date("2021-01-01") }); // prettier-ignore
+    const thread2 = dummyThreadData({ roomId, createdAt: new Date("2021-01-02") }); // prettier-ignore
+    const thread3 = dummyThreadData({ roomId, createdAt: new Date("2021-01-03") }); // prettier-ignore
 
-    const inboxNotificationsPageOne = [
+    const inboxNotificationsPage1 = [
       dummyThreadInboxNotificationData({
         roomId,
-        threadId: threadOne.id,
+        threadId: thread1.id,
+        notifiedAt: new Date("2021-01-03"),
```

Comment:
This unit test could be somewhat unstable, occasionally not passing. I already fixed this on another branch. It's unrelated to this PR, but I'm cherry-picking this fix here now, to ensure the unit tests are fixed on `main` ASAP.